msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/ResetPasswordStates.kt

```diff
@@ -66,11 +67,13 @@ import kotlinx.coroutines.withContext
  */
 class ResetPasswordCodeRequiredState internal constructor(
     override val continuationToken: String,
+    private val username: String,
     private val config: NativeAuthPublicClientApplicationConfiguration
 ) : BaseState(continuationToken), State, Parcelable {
     private val TAG: String = ResetPasswordCodeRequiredState::class.java.simpleName
 
     constructor(parcel: Parcel) : this(
+        parcel.readString() ?: "",
```

Comment:
The `writeToParcel` method for this class should be updated.

Reply:
Done